libs/scrap/src/android/ffi.rs

```diff
@@ -49,10 +49,11 @@ impl FrameRaw {
         self.enable = value;
     }
 
-    fn update(&mut self, data: *mut u8) {
+    fn update(&mut self, data: *mut u8, len: usize) {
         if self.enable.not() {
             return;
         }
+        self.len = len;
```

Comment:
Big threading problem here, len and data are not synced. And also no guarentee of the lifetime of data. I will note this in https://github.com/rustdesk/rustdesk/issues/4118